.build/rvk/data.k

```diff
@@ -401,22 +414,37 @@ module EVM-DATA
 
 // -   `#newAddr` computes the address of a new account given the address and nonce
 //     of the creating account.
+// -   `#sender` computes the sender of the transaction from its data
+//     and signature.
 
 
     syntax Int ::= #newAddr ( Int , Int ) [function]
  // ------------------------------------------------
     rule #newAddr(ACCT, NONCE) => #addr(#parseHexWord(Keccak256(#rlpEncodeLength(#rlpEncodeWord(ACCT) +String #rlpEncodeWord(NONCE), 192))))
 
-    syntax String ::= #rlpEncodeLength ( String , Int )          [function]
-                    | #rlpEncodeLength ( String , Int , String ) [function, klabel(#rlpEncodeLengthAux)]
-                    | #rlpEncodeWord ( Int )                     [function]
- // -----------------------------------------------------------------------
-    rule #rlpEncodeWord(0) => "\x80"
-    rule #rlpEncodeWord(WORD) => chrChar(WORD) requires WORD >Int 0 andBool WORD <Int 128
-    rule #rlpEncodeWord(WORD) => #rlpEncodeLength(#unparseByteStack(#asByteStack(WORD)), 128) requires WORD >=Int 128
-    rule #rlpEncodeLength(STR, OFFSET) => chrChar(lengthString(STR) +Int OFFSET) +String STR requires lengthString(STR) <Int 56
-    rule #rlpEncodeLength(STR, OFFSET) => #rlpEncodeLength(STR, OFFSET, #unparseByteStack(#asByteStack(lengthString(STR)))) requires lengthString(STR) >=Int 56
-    rule #rlpEncodeLength(STR, OFFSET, BL) => chrChar(lengthString(BL) +Int OFFSET +Int 55) +String BL +String STR
+    syntax Int ::= #sender ( Int , Int , Int , Int , Int , String , Int , WordStack , WordStack ) [function]
+                 | #sender ( String , Int , String , String )                                     [function, klabel(#senderAux)]
+ // ----------------------------------------------------------------------------------------------------------------------------
+    rule #sender(TN, TP, TG, TT, TV, DATA, TW, TR, TS)
+      => #sender(#unparseByteStack(#parseHexBytes(Keccak256(#rlpEncodeLength(#rlpEncodeWordStack(TN : TP : TG : TT : TV : .WordStack) +String #rlpEncodeString(DATA), 192)))), TW, #unparseByteStack(TR), #unparseByteStack(TS))
```

Comment:
using rlpEncodeWordStack here is not necessarily wise because TT is a 20-byte byte array, not an integer. It's a bug in this implementation that I have fixed in my next PR.